app/app.go

```diff
@@ -617,6 +625,20 @@ func New(
 		app.EvmKeeper.EthClient = ethclient.NewClient(rpcclient)
 	}
 
+	if app.evmRPCConfig.LiveEVMTracer != "" {
+		// PR_REVIEW_NOTE: So I moved this code from `ProcessBlock` and there, I had access to `ctx` so the code was actually looking
```

Comment:
we will refactor our ChainID access so that it doesn't require ctx. Will update here once done

Reply:
@maoueh we've merged https://github.com/sei-protocol/sei-chain/pull/1548 so that you can now get ChainID without ctx